src/OnboardingSPA/components/SocialMediaForm/index.js

```diff
@@ -175,7 +208,7 @@ const SocialMediaForm = ({ socialData, setSocialData, setIsValidSocials }) => {
                         <div className="social-form__label_icon" style={{ backgroundImage: `var(--${SocialMediaSites[social]}-icon)` }} />
                         <div className="social-form__label_name">{__(toTitleCase(SocialMediaSites[social]), 'wp-module-onboarding')}</div>
                     </label>
-                    <Tooltip content={activeError.includes(SocialMediaSites[social]) ? `Please enter a valid ${SocialMediaSites[social]} URL` : 'hide'} direction="top">
+                    <Tooltip content={activeError.includes(SocialMediaSites[social]) ? ( (social == 'TWITTER') ? `Please enter a valid ${SocialMediaSites[social]} URL / username` : `Please enter a valid ${SocialMediaSites[social]} URL` ): 'hide'} direction="top">
```

Comment:
Can we also refactor this to a function? if we start supporting facebook usernames then all the checks will make this statement just longer.

Reply:
done